pkg/controller/constrainttemplate/constrainttemplate_controller_test.go

```diff
@@ -540,18 +540,14 @@ func TestReconcile(t *testing.T) {
 			logger.Error(err, "create cstr")
 			t.Fatal(err)
 		}
-		err = retry.OnError(testutils.ConstantRetry, func(_ error) bool {
-			return true
-		}, func() error {
-			// check if vapbinding resource exists now
-			vapBinding := &admissionregistrationv1beta1.ValidatingAdmissionPolicyBinding{}
```

Comment:
is there a separate test to make sure admissionregistrationv1beta1 vapb is tested? we should not remove old tests when adding new tests especially if it reduces test coverage.

Reply:
Yes, for example this - https://github.com/open-policy-agent/gatekeeper/pull/3479/files#diff-acfa8801c77d031956edb3d5c1078fe12346ff32dcda82fc3ce60a7ab9c013d3R453, and several others tests are for `v1beta1` VAP/VAPB, for `v1` the test names include `with v1`.